spec/features/support/providers/courses/viewing_providers_courses_spec.rb

```diff
@@ -6,26 +6,51 @@
   scenario 'Provider is discarded' do
     given_i_am_authenticated_as_an_admin_user
     and_there_is_a_discarded_provider_with_courses
-    when_i_visit_the_support_provider_courses_index_page
+    when_i_visit_the_support_discarded_provider_courses_index_page
     then_i_am_redirected_to_the_providers_page
   end
 
+  scenario 'viewing course status tags' do
+    given_i_am_authenticated_as_an_admin_user
+    and_there_is_a_provider_with_courses
+    when_i_visit_the_support_provider_courses_index_page
+    then_i_should_see_the_open_status
+  end
+
   private
 
+  def then_i_should_see_the_open_status
+    within('td.govuk-table__cell.status') do
+      expect(page).to have_css('strong.govuk-tag.govuk-tag--turquoise', text: 'Open')
+    end
+  end
+
   def given_i_am_authenticated_as_an_admin_user
     given_i_am_authenticated(user: create(:user, :admin))
   end
 
+  def discarded_provider
+    @provider ||= create(:provider, id: 1, courses: [build(:course)], discarded_at: Time.zone.now)
+  end
+
   def provider
-    @provider ||= create(:provider, courses: [build(:course)], discarded_at: Time.zone.now)
+    @provider ||= create(:provider, id: 2, courses: [build(:course, :published, :open)])
   end
 
-  def and_there_is_a_discarded_provider_with_courses
+  def and_there_is_a_provider_with_courses
     provider
   end
 
+  def and_there_is_a_discarded_provider_with_courses
+    discarded_provider
+  end
+
   def when_i_visit_the_support_provider_courses_index_page
-    support_provider_courses_index_page.load(recruitment_cycle_year: Settings.current_recruitment_cycle_year, provider_id: provider.id)
+    support_provider_courses_index_page.load(recruitment_cycle_year: Settings.current_recruitment_cycle_year, provider_id: 2)
```

Comment:
Using ints for ids doesn't look good. Can we try to use references instead?

Reply:
I've used a reference for this now but in order to do that i've kept the `and_there_is_a_discarded_provider_with_courses` method as it is, so the `discarded_provider` can be referenced.